pkg/provisioners/helmapplications/vcluster/provisioner.go

```diff
@@ -19,7 +19,6 @@ package vcluster
 
 import (
 	"context"
```

Comment:
This is a legitimate error because you've deleted something you shouldn't have...

Reply:
friggin IDE!

Reply:
Wait, am I missing something? Itt's only a new line been removed? Is there a formatting requirement here?

Reply:
ok, evidently it is!! 

Christ, stupid IDE formatting - ok resolving this. I'll still get a linter bump pushed in across all repos as it doesn't hurt either way